CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## Version 1.7.0
```

Comment:
I based versioning off the version used by the gemspec. I opted for a minor version bump, based on my understanding of semantic versioning. Although, given these changes are _technically_ breaking, you could make the case for version `2.0.0` instead of `1.7.0`. Thoughts @timbreitkreutz?

Reply:
Our repo is a fork. We don't control the condensation gem.

https://rubygems.org/gems/condensation

Our fork currently has one other fork, which belongs to @wstrinz. If there is some way to pull metrics of usage I'm happy to do that. Otherwise I only have the forks as an indicator

Reply:
Yeah even though it's a fork we seem to have set it up as a public repo...  Maybe the spec change isn't important.  Perhaps the best way forward is to fork this into a private repo inside of the DripEmail GitHub account, and do our new work there?

Reply:
That seems like a reasonable idea!